splice_machine/src/main/java/com/splicemachine/derby/impl/sql/compile/costing/V1ScanCostEstimator.java

```diff
@@ -127,8 +127,9 @@ public void generateCost(long numFirstIndexColumnProbes) throws StandardExceptio
         double closeLatency = scc.getCloseLatency();
         double localLatency = scc.getLocalLatency();
         double remoteLatency = scc.getRemoteLatency();
+        // IndexPrefixIteration is not currently parallel, so penalize by a factor of the parallelism.
         double remoteCost = (openLatency + closeLatency) +
-                (numFirstIndexColumnProbes*2)*remoteLatency*(1+colSizeFactor/1024d) +
+                (numFirstIndexColumnProbes*2*scc.getParallelism())*remoteLatency*(1+colSizeFactor/1024d) +
```

Comment:
are these changes to scan cost relevant to this PR, if not, could you please revert them and create a special PR for them? (this would be nice so we can adapt to these changes in V2 cost model when we bring DB-11931 in.

Reply:
This is relevant because the test case runs slowly without it.  It is meant only as a temporary change, and I would revert it as soon as I'm done with DB-11930 (which I'm currently working on).  If you don't mind, I'd rather spend the time working on the root cause, DB-11930, instead of spending extra time to separate this temporary change out when it would be reverted soon anyway.